src/assets/alignmentD3Tests.js

```diff
@@ -314,17 +353,29 @@ var getMisalignmentCountCaller = function(
  * @param {Array}  collection - HTMLCollection of features
  * @param {Function} getMisalignmentCountFunc -
  * passes either getXMisalignmentCount or getYMisalignmentCount
+ * @param {Function} getFeatureValueFunc - getFeatureValueInteger or
+ * getFeatureValueMinutes
+ * @param {Function} getTickValueFunc - getTickValueInteger,
+ * getTickValueMinutes, or getTickValueMonths
  * @returns {Boolean} true if no misalignments are counted
```

Comment:
It is a minor detail, but I always put a space before the `@returns` so it stands out from the parameters.